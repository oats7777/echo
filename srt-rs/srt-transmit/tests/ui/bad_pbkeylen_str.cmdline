["srt://:4000?pbkeylen=twelve&passphrase=hellooooo" ,  "udp://127.0.0.1:4000"]