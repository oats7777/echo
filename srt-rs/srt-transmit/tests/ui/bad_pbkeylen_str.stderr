Invalid settings detected: Failed to parse key length: invalid digit found in string

See srt-transmit --help for more info